week2/homework/src/index.js

```diff
@@ -1,3 +1,82 @@
-'use strict';
+'use strict:';
 
-// TODO: Write the homework code in this file
+const fs = require('fs');
+const { promisify } = require('util');
+const dataToNewLine = data => `\n${data}`;
+
+const writeFile = promisify(fs.writeFile);
+const readfile = promisify(fs.readFile);
+
+const filePath = './ToDoList.json';
+
+const getAllToDos = () => {
+  return readfile(filePath, 'utf8')
+    .then(data => JSON.parse(data))
+    .catch(err => console.log(err));
+};
+
+const updateTodo = async (id, newTitle) => {
+  const { todos } = await getAllToDos();
```

Comment:
just for my understanding: why did you decide to use async here instead of then/catch?

Reply:
I am not sure I have to make promises and I choose async function.